client-admin/Dockerfile

```
@@ -1,20 +1,16 @@
 # Requires a very specific version due to package bugs and version compatibility
-FROM node:6.10.3
+FROM node:6.10.3-alpine
 
 ARG GIT_HASH
 
 WORKDIR /app
 
-RUN npm install -g gulp
+RUN apk add git --no-cache
 
-ADD package*.json npm-shrinkwrap.json ./
-# envvar hides core-js sponsorship message that errors in postinstall.
-# See: https://github.com/zloirock/core-js/pull/799#issuecomment-611437614
-ENV ADBLOCK=1
-RUN npm install
+COPY package*.json npm-shrinkwrap.json ./
+RUN npm install --silent
```

```suggestion
RUN npm install
```

Could we avoid silent? Esp for newcomers, I feel that output helps wayfinding within install process and helps troubleshoot when things go wrong :)

---

It's pretty noisy and often not very helpful. But sure. We can put --silent back if it gets to be too much.

---

Thanks. It has def helped me know where hangs have been happening :)